examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp

```diff
@@ -241,7 +241,8 @@ void drawTopology()
                     int y = 0;
                     int x = 0;
                     getyx(topoPad, y, x);
-                    if (x >= topoPadmaxX) wprintw(topoPad, "\n");
+                    if (x >= topoPadmaxX)
+                        wprintw(topoPad, "\n");
                     wprintw(topoPad, "0%o[%d] ", mesh.addrList[j].address, mesh.addrList[j].nodeID);
```

Comment:
I think this is how we got stung before in RF24 repo when some conditional blocks without curly brackets were re-formatted (and there was no blank line afterward). I want to test this on hardware before merging.

Reply:
Nice catch, I really dislike that format with no brackets.

Reply:
clang-format has a [config option for these "short blocks"](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#allowshortifstatementsonasingleline), but it seems to only deem a block as short if it contains 1 keyword statement (`return`/`break`/`continue`).
I have this set to `Never` in the config:
https://github.com/nRF24/RF24Mesh/blob/9ed627323e39e648fe431dc914868ffb0c6d0199/.clang-format#L22
I think I can set it to `AllIfsAndElse` to avoid reformatting these short if blocks.

As for the curly brackets, there's a way to tell clang-format to `Always` [add brackets when not present after a control statement](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#bracewrapping). Currently, I have it set to do this only when the condition spans multiple lines.
https://github.com/nRF24/RF24Mesh/blob/9ed627323e39e648fe431dc914868ffb0c6d0199/.clang-format#L34